lmdeploy/model.py

```diff
@@ -1324,6 +1324,45 @@ def match(cls, model_path: str) -> Optional[str]:
             return 'internvl-zh-hermes2'
 
 
+@MODELS.register_module(name='phi-3')
+class Phi3Instruct(BaseChatTemplate):
+    """Chat template of InternLM model."""
+
+    def __init__(
+            self,
+            system='<|system|>\n',
+            meta_instruction="""You are a helpful AI assistant.""",  # noqa: E501
+            eosys='<|end|>\n',
+            user='<|user|>\n',
+            eoh='<|end|>\n',
+            assistant='<|assistant|>:',
```

Comment:
Shall we change `:` to `\n` according to the [config](https://huggingface.co/microsoft/Phi-3-mini-128k-instruct/blob/8a362e755d2faf8cec2bf98850ce2216023d178a/tokenizer_config.json#L119)?
```
"chat_template": "{{ bos_token }}{% for message in messages %}{% if (message['role'] == 'user') %}{{'<|user|>' + '\n' + message['content'] + '<|end|>' + '\n' + '<|assistant|>' + '\n'}}{% elif (message['role'] == 'assistant') %}{{message['content'] + '<|end|>' + '\n'}}{% endif %}{% endfor %}",
 ```